OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -85,10 +87,29 @@ public void InvokeCommand(string name, string arg)
 					IssueDevCommand(world, "DevEnableTech");
 					IssueDevCommand(world, "DevFastCharge");
 					IssueDevCommand(world, "DevGiveCash");
+					IssueDevCommand(world, "DevLevelUp");
 					break;
 
 				case "crash":
 					throw new DevException();
+				
+				case "levelup":
+					var level = 0;
```

Comment:
No, this is a bad plan. Methods taking `out` parameters *must* either write them (even in the TryParse "failure" case) or throw. The original value does not survive. int.TryParse sets its `out` parameter to zero on failure.